docs/source/examples/Widget Styling.ipynb

```diff
@@ -169,7 +169,42 @@
     }
    ],
    "source": [
-    "from ipywidgets import HBox, Label, IntSlider\n",
+    "from ipywidgets import IntSlider\n",
+    "\n",
+    "IntSlider(description='A too long description')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can set the length of the description are to match the description, the widget's length is shorter in presence of a description. This because the description is added *inside* of the widget's total length."
```

Comment:
How about:

You can change the length of the description to fit the description text. However, this will make the widget itself shorter. You can change both by adjusting the description width and the widget width using the widget's style.